share/store/file/axis_half.go

```diff
@@ -0,0 +1,19 @@
+package file
+
+import (
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-node/share/shwap"
+)
+
+type AxisHalf struct {
+	Shares   []share.Share
+	IsParity bool
```

Comment:
Lets just the shwap sides. It its much clearer to have one way of doing that in the code base

Reply:
What is left side of columnt?

Reply:
Single type > semantical correctness(that can be documented)

Reply:
It will be confusing to have Columns types to be names as shwap.Row and to have left and right sides. I would rather use file.AxisHalf type in shwap pkg for shwap.HalfRow, as it is the type that describes both rows and columns.

Reply:
But we can't use AxisHalf in shwap because its not part of the spec